### PR TITLE
Portuguese translation benchmarks

### PR DESCRIPTION
**Pull request by Serrones**
`benchmark.md` translated to Brazilian Portuguese

### CODE REVIEW

File: docs/pt/docs/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Comparações
+
+As comparações independentes da TechEmpower mostram as aplicações **FastAPI** rodando com Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">um dos _frameworks_ Python mais rápidos disponíveis</a>, somente atrás dos próprios Starlette e Uvicorn (utilizados internamente pela FastAPI). (*)
```
Comment (marcosmmb) [inline docs/pt/docs/benchmarks.md:3]:
```suggestion
As comparações independentes da TechEmpower mostram as aplicações **FastAPI** rodando com Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">um dos _frameworks_ Python mais rápidos disponíveis</a>, somente atrás dos próprios Starlette e Uvicorn (utilizados internamente pelo FastAPI). (*)
```

File: docs/pt/docs/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Mas quando se checa _benchmarks_ e comparações você deveria ter o seguinte em mente. 
+
+## Comparações e velocidade
+
+Quando você verifica os _benchmarks_, é comum observar algumas ferramentas de diferentes tipos, mas equivalentes.
+
+Especificamente para observar Uvicorn, Starlette e FastAPI comparados juntos (entre muitas outras ferramentas).
```
Comment (marcosmmb) [inline docs/pt/docs/benchmarks.md:11]:
```suggestion
Especificamente, observa-se Uvicorn, Starlette e FastAPI comparados juntos (entre muitas outras ferramentas).
```

File: docs/pt/docs/benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Comparações
+
+As comparações independentes da TechEmpower mostram as aplicações **FastAPI** rodando com Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">um dos _frameworks_ Python mais rápidos disponíveis</a>, somente atrás dos próprios Starlette e Uvicorn (utilizados internamente pela FastAPI). (*)
+
+Mas quando se checa _benchmarks_ e comparações você deveria ter o seguinte em mente. 
+
+## Comparações e velocidade
+
+Quando você verifica os _benchmarks_, é comum observar algumas ferramentas de diferentes tipos, mas equivalentes.
```
Comment (marcosmmb) [inline docs/pt/docs/benchmarks.md:9]:
```suggestion
Ao verificar os _benchmarks_, é comum observar algumas ferramentas de diferentes tipos comparadas como equivalentes.
```

File: docs/pt/docs/benchmarks.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Especificamente para observar Uvicorn, Starlette e FastAPI comparados juntos (entre muitas outras ferramentas).
+
+Quanto mais simples o problema resolvido pela ferramenta, melhor a performance que ela terá. E a maioria dos _benchmarks_ não testam as características adicionais fornecidas pela ferramenta.
+
+A hierarquia segue assim:
+
+* **Uvicorn**: um servidor ASGI
+    * **Starlette**: (utiliza Uvicorn) um _microframework web_
+        * **FastAPI**: (utiliza Starlette) um microframework API com várias características adicionais para construção de APIs, com validação de dados, etc.
```
Comment (marcosmmb) [inline docs/pt/docs/benchmarks.md:19]:
```suggestion
        * **FastAPI**: (utiliza Starlette) um _microframework_ de API com vários recursos adicionais para construção de APIs, com validação de dados, etc.
```

File: docs/pt/docs/benchmarks.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+* **Uvicorn**:
+    * Terá a melhor performance, já que ele não tem muito código extra a parte do servidor em si.
```
Comment (marcosmmb) [inline docs/pt/docs/benchmarks.md:22]:
```suggestion
    * Terá a melhor performance, já que ele não tem muito código extra além do servidor em si.
```

File: docs/pt/docs/benchmarks.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    * Você não consegue escrever uma aplicação em Uvicorn diretamente. Isso significa que seu código deveria conter, mais ou menos, todo o código fornecido pelo Starlette (ou **FastAPI**). E se você fizesse isso, sua aplicação final poderia ter o mesmo peso do que utilizar um _framework_ que minimiza o código de sua aplicação e _bugs_.
```
Comment (marcosmmb) [inline docs/pt/docs/benchmarks.md:23]:
```suggestion
    * Você não conseguiria escrever uma aplicação em Uvicorn diretamente. Isso significa que seu código deveria conter, mais ou menos, todo o código fornecido pelo Starlette (ou **FastAPI**). E se você fizesse isso, sua aplicação final poderia ter a mesma sobrecarga que utilizar um _framework_ que minimiza o código e _bugs_ da sua aplicação.
```

File: docs/pt/docs/benchmarks.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    * Se você quer fazer comparações com o Starlette, compare com Sanic, Flask, Django, etc. _Frameworks Web_ (ou _microframeworks_).
+* **FastAPI**:
+    * Do mesmo modo que Starlette utiliza Uvicorn e não pode ser mais rápido que ele, **FastAPI** utiliza o Starlette, então não tem como ser mais rápido do que o Starlette.
+    * FastAPI fornece mais recursos acima do Starlette. Recursos que você quase sempre precisará quando construir APIs, como validação de dados e serialização. E utilizando eles, você terá uma documentação automática de graça (a documentação automática nem sequer adiciona peso para rodar as aplicações, ela é gerada na inicialização).
+    * Se você nunca utilizou FastAPI mas utilizou diretamente o Starlette (ou outra ferramenta, como Sanic, Flask, Responder, etc) você teria que implementar toda validação de dados e serialização por conta. Então, sua aplicação final poderia ainda ter o mesmo peso como se fosse desenvolvido com FastAPI. E em muitos casos, a validação de dados e serialização é o grande pedaço de código escrito em aplicações.
```
Comment (marcosmmb) [inline docs/pt/docs/benchmarks.md:32]:
```suggestion
    * Se você nunca utilizou FastAPI mas utilizou diretamente o Starlette (ou outra ferramenta, como Sanic, Flask, Responder, etc) você teria que implementar toda validação de dados e serialização por conta. Então, sua aplicação final poderia ainda ter a mesma sobrecarga como se fosse desenvolvida com FastAPI. Em muitos casos, a validação de dados e serialização é o maior pedaço de código escrito em aplicações.
```

File: docs/pt/docs/benchmarks.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    * Então, ao utilizar FastAPI você estará economizando tempo de desenvolvimento, evitar _bugs_, linhas de código, e você provavelmente terá a mesma performance (ou melhor) do que não utilizá-lo (já que você teria que implementar tudo em seu código).
```
Comment (marcosmmb) [inline docs/pt/docs/benchmarks.md:33]:
```suggestion
    * Então, ao utilizar o FastAPI você estará economizando tempo de desenvolvimento, evitará _bugs_, linhas de código, e você provavelmente terá a mesma performance (ou melhor) do que não utilizá-lo (já que você teria que implementar tudo isso em seu código).
```

File: docs/pt/docs/benchmarks.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    * Se você quer fazer comparações com o FastAPI, compare com uma aplicação _web framework_ (ou conjunto de ferramentas) que forneça validação de dados, serialização e documentação, como Flask-apispec, NestJS, Molten, etc. _Frameworks_ com validação de dados automática inegrada, serialização e documentação.
```
Comment (marcosmmb) [inline docs/pt/docs/benchmarks.md:34]:
```suggestion
    * Se você quer fazer comparações com o FastAPI, compare com um _framework_  (ou conjunto de ferramentas) para aplicações _web_ que forneça validação de dados, serialização e documentação, como Flask-apispec, NestJS, Molten, etc. _Frameworks_ com validação de dados automática, serialização e documentação integradas.
```

**Review by marcosmmb — COMMENTED** [review]
Several adjustments and fixes

**Comment by Serrones** [comment]
Thanks a lot @marcosmmb . All suggestions were included.

**Review by cassiobotaro — APPROVED** [review]

**Review by marcosmmb — APPROVED** [review]
LGTM

**Comment by jfunez** [comment]
LGTM

**Review by jfunez — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome! Thank you @Serrones ! :rocket: :coffee: 

And thanks for the reviews @jfunez , @cassiobotaro , and @marcosmmb ! :bow: :cake:

**Pull request merged**